frontend/src/components/widgets/CameraInput/CameraInput.tsx

```diff
@@ -256,10 +256,15 @@ class CameraInput extends React.PureComponent<Props, State> {
   public componentDidUpdate = (prevProps: Props): void => {
     const { element, widgetMgr } = this.props
 
+    // TODO(vdonato): Rework this now that there's a short window where the app
+    // may reconnect to the server without losing its uploaded files. Just
+    // removing this seemed to work, but I'm not entirely sure if it's the
+    // correct fix.
+    //
```

Comment:
Ah, I was referring to the if statement immediately below the comment. Changed the comment to clarify this (in a commit pushed directly to the feature branch since it doesn't seem worth making a separate PR for that)